### ISSUE TITLE
Feature request: Cut & Paste tabs between groups

### ISSUE THREAD

**Issue by AfroThundr3007730** [Drive4ik/simple-tab-groups#396]
Not sure how feasible this is, but it would be great if the extension supported cut and pasting of tabs between groups. This could significantly speed up tab management functionality.

I envision a workflow similar to the following:

  - Select some tabs from a group that you want to move
  - Hit Ctrl+X to "cut" those tabs out of the current group
  - Click the header of the group you want to drop the tabs in
  - Hit Ctrl+V to "paste" the cut tabs into the chosen group
  - These could also be in the right-click menu on the group

Other considerations for fully keyboard-controlled tab management:

  - Maybe a "copy" function (Ctrl+C) to copy/duplicate a tab or tab set between groups
  - Maybe also add arrow key navigation to move between the groups (sort of like a cursor, or selection/highlight box)
    - Pasting on the highlighted/selected group would drop the tabs at the end
    - Spacebar could put the cursor into the group, allowing for a cursor to move among the tabs
    - Pasting would put the tabs at the cursor position in the group
    - Esc could return to the group navigation mode
  - Not sure if you can intercept Ctrl+F and redirect it to the tab search box...

**Comment by Drive4ik** [comment]
And what does this option not suit: tab selection -> context menu -> move to group? (In popup window)

**Comment by AfroThundr3007730** [comment]
That method would require constantly switching between the groups and lots of scrolling on the tab bar to find/select the tabs. The tab manager view is really handy for doing all of this in one place.

In my use case, I currently have ~35 tab groups, most with 60+ tabs (and several with 200+). Exposing that same context menu in the tab manager view would make organizing them easier (and so would the keybindings).

The suggestion for adding keyboard-only navigation would just complete the feature set.

**Comment by Drive4ik** [comment]
Yes, I agree with you about the context menu in manage groups, but I meant the context menu in the popup window of addon

**Comment by AfroThundr3007730** [comment]
Oh cool, I hadn't noticed that context menu existed. It's pretty much the exact same thing I was requesting for the manage groups view. :)

The reason to also add it to the manage groups view at this point, would be for those who like to organize their tab groups visually. A bit redundant, but the entire manage groups view is functionally redundant to the extension popup menu anyway.
